server/src/main/java/org/apache/cassandra/sidecar/metrics/server/AuthMetrics.java

```diff
@@ -32,17 +33,23 @@ public class AuthMetrics
     private static final String DOMAIN = SERVER_PREFIX + ".Auth";
     public final NamedMetric<Counter> jwtPemRefreshFailures;
     public final NamedMetric<Counter> jwtPemRefreshSuccesses;
+    public final NamedMetric<Timer> authorizationTime;
```

Comment:
I think we should document that this authorization time is only taking into account authorizations that are not cached